subsys/net/l2/ppp/misc.c

```diff
@@ -98,6 +98,12 @@ void ppp_change_phase(struct ppp_context *ctx, enum ppp_phase new_phase)
 	validate_phase_transition(ctx->phase, new_phase);
 
 	ctx->phase = new_phase;
+
+	if (ctx->phase == PPP_DEAD)
```

Comment:
Please add `{}` even for one liners.
Can you use `else` here as the phase can only have one value so 
```
if (....DEAD) {

} else if (...RUNNING) {

}
```